src/vs/workbench/contrib/terminal/browser/media/terminal.css

```diff
@@ -93,6 +93,7 @@
 	display: block;
 }
 
+.monaco-workbench .pane-body.integrated-terminal .terminal-group .monaco-split-view2.horizontal .split-view-view:first-child .xterm,
```

Comment:
I think it would be better to have this at the view level since it's a global terminal thing, similar to bold:

https://github.com/microsoft/vscode/blob/210f772b0174eb7c3d3ebb17ddbba67d887e5fe2/src/vs/workbench/contrib/terminal/browser/media/terminal.css#L165-L172

Reply:
Not sure what you're proposing 🤔

Reply:
need to have access to the `split-view-view` in order to know if it's the `first-child`

Reply:
Oh the bold rule doesn't seem to be used anymore 🤔 

I mean moving assigning the class into `terminalView`, so it's set/unset on only a single element